### PR TITLE
SE-1381: add env var

### PR DESCRIPTION
**Pull request by slemasne**
# Pull Request Checklist

- [x] Read the [contributing guidelines](../docs/CONTRIBUTING.md)
- [x] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

**Comment by jyaffe** [comment]
Looks good

**Review by jyaffe — APPROVED** [review]
Looks good!

File: .github/workflows/build-and-test.yaml [hunk 1]
```diff
@@ -42,6 +42,7 @@ jobs:
           FBN_CLIENT_SECRET: ${{ secrets.DEVELOP_FBN_CLIENT_SECRET }}
           FBN_LUSID_API_URL: ${{ secrets.DEVELOP_FBN_LUSID_API_URL }}
           FBN_APP_NAME: ${{ secrets.DEVELOP_FBN_CLIENT_ID }}
+          FBN_LUSID_PAT: ${{ secrets.DEVELOP_FBN_ACCESS_TOKEN }}
```
Comment (rizwansaeed) [inline .github/workflows/build-and-test.yaml:45]: i think this should be `FBN_LUSID_ACCESS_TOKEN`

Comment (slemasne) [reply .github/workflows/build-and-test.yaml]: Thanks. Updated.

**Pull request merged**